crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -172,6 +171,8 @@ public Response insert(InsertionRequest req) {
                 METRICS.debug("insert: {}",ctx.measure);
             }
             Error.pop();
+            markRequestErrors(response.getErrors(), metricCtx);
+            metricCtx.endRequestMonitoring();
```

Comment:
Fixed as per suggestion. Mediator will only track the individual bulkrequests around their invocations.